docs/whatsnew/3.1.rst

```diff
@@ -35,7 +35,14 @@ We have bumped the minimum version of several packages we depend on; these are t
 - astropy >= 4.2
 - numpy >= 1.17.0
 
-.. _whatsnew-3.1-contributors:
+.. _whatsnew-3.1-draw-limb:
+
+New limb drawing function
+=========================
+The solar limb as seen from an arbitrary observer coordinate can now be drawn using the :func:`sunpy.visualization.draw_limb`
+function.
```

Comment:
Elaborate on what the limb can now be drawn (i.e., a `WCSAxes` instance, but less technical).

Reply:
I have attempted to address this, but better wording suggestions very welcome...